_data/current.yml

```diff
@@ -218,6 +218,14 @@
   url: http://www.ministryoftesting.com/2016/12/testbash-coming-germany/
   twitter: ministryoftest
   status: CFP is open
+  
+- name: Tricentis Accelerate 2017
+  location: Vienna, Austria
+  dates: "October 16-17, 2017"
+  url: https://www.tricentis.com/accelerate2017/
+  status: CFP & Registration is open
```

Comment:
Ideally this would be a single status like CFP is open. Registration being open is implied.